docs/layouts/shortcodes/generated/yarn_config_configuration.html

```diff
@@ -152,6 +152,18 @@
             <td>String</td>
             <td>The provided usrlib directory in remote. It should be pre-uploaded and world-readable. Flink will use it to exclude the local usrlib directory(i.e. usrlib/ under the parent directory of FLINK_LIB_DIR). Unlike yarn.provided.lib.dirs, YARN will not cache it on the nodes as it is for each application. An example could be hdfs://$namenode_address/path/of/flink/usrlib</td>
         </tr>
+        <tr>
+            <td><h5>yarn.rolled-logs.exclude-pattern</h5></td>
+            <td style="word-wrap: break-word;">"hadoopfs"</td>
+            <td>String</td>
+            <td>Java regular to exclude certain log files from rolling log aggregation. Log files matching the defined exclude pattern will be ignored during aggregation. If a log file matches both the include and exclude patterns, the exclude pattern takes precedence and the file will be excluded from aggregation.</td>
```

Comment:
s/regular to/regular expression to/

Reply:
> If a log file matches both the include and exclude patterns

Optional: I would add some warning when a file hits such case. It's hard to know what happened and why without any log.

Reply:
To provide a useful warning we'd need validation logic if the 2 regex can match to the same input. That's a [non-trivial](https://stackoverflow.com/questions/22646997/detecting-if-two-regexes-could-possibly-match-the-same-string/22665097#22665097), possibly quite expensive operation, so i don't think it's worth it.